phangsPipeline/utilsImages.py

```diff
@@ -0,0 +1,160 @@
+# Routines relates to python (not CASA) image handling.
+
+import numpy as np
+import os
+import re
+import warnings
+import math
+
+import astropy.units as u
+from astropy.io import fits
+from astropy.wcs import WCS
+from astropy.coordinates import SkyCoord
+
+# ------------------------------------------------------------------------
+# Make new headers
+# ------------------------------------------------------------------------
+
+def make_simple_header_from_box(
+        tlc_coord = None, brc_coord = None,
+        pix_scale_deg = None, pad_arcsec = 0.0):
```

Comment:
Suggest passing these as angular quantities rather than having the kwarg define the unit.

for comparisons,
```
pix_scale = pix_scale.to(u.deg)
```
and allow astropy's builtin unit equivalence checks to kick in.